### PR TITLE
Horizontally flip player.png

### LINKED ISSUE #4329: 2D Player texture on front view looks like mirrored. Instead of normal.
2d skin bug
I'm on Minimal Development Test subgame, for testing 2D textures. It's a bug on 2D player front texture, because, 2D player front texture looks like a mirrored in-game. Must be normal look...

### PR DESCRIPTION
**Pull request by SmallJoker**
Fixes #4329

Successfully tested with MSVC.

### CODE REVIEW

File: src/content_cao.cpp [hunk 1]
```diff
@@ -839,54 +839,51 @@ void GenericCAO::addToScene(scene::ISceneManager *smgr,
...
-		mesh->addMeshBuffer(buf);
-		buf->drop();
+			scene::IMeshBuffer *buf = new scene::SMeshBuffer();
+			u8 li = m_last_light;
+			video::SColor c(255,li,li,li);
+			video::S3DVertex vertices[4] = {
+				video::S3DVertex(-dx, -dy, 0, 0,0,0, c, 1,1),
+				video::S3DVertex( dx, -dy, 0, 0,0,0, c, 0,1),
+				video::S3DVertex( dx,  dy, 0, 0,0,0, c, 0,0),
+				video::S3DVertex(-dx,  dy, 0, 0,0,0, c, 1,0),
```
Comment (rubenwardy) [inline src/content_cao.cpp]: Note to reviewers, the code has changed here ^

**Comment by rubenwardy** [comment]
It looks like all you've changed is the front side

**Comment by SmallJoker** [comment]
Yes, assuming the front image is "the correct" image. It could be the other way done but using the front seems more logical.

File: src/content_cao.cpp [hunk 2]
```diff
@@ -839,54 +839,51 @@ void GenericCAO::addToScene(scene::ISceneManager *smgr,
 			setBillboardTextureMatrix(m_spritenode,
 					txs, tys, 0, 0);
 		}
-	}
-	else if(m_prop.visual == "upright_sprite") {
+	} else if(m_prop.visual == "upright_sprite") {
```
Comment (rubenwardy) [inline src/content_cao.cpp]: `if (`

**Comment by est31** [comment]
Would have been nicer to have it in two commits, one for style cleanup, the other one that changes the actual lines, but that's only a detail. Looks good :+1: .

**Comment by est31** [comment]
@paramat I assume you approved this PR as well? I'll add the according label then. Remove it if you labeled without approving (can't tell as you only labeled).

**Comment by paramat** [comment]
Oops forgot to :+1:

**Comment by paramat** [comment]
8a98cbda8417c713e03d1ac0be80e88cd2f08a0c

**Pull request closed without merging**